doc/configcommands.dsv

```diff
@@ -129,7 +129,7 @@ run-on-startup||<list of operations>||n/a||Specifies one or more <<_newsboat_ope
 save-path||<path-to-directory>||~/||The default path where articles shall be saved to. If an invalid path is specified, the current directory is used.||save-path "~/Saved Articles"
 scrolloff||<number>||0||Keep the configured number of lines above and below the selected item in lists. Configure a high number to keep the selected item in the center of the screen.||scrolloff 5
 search-highlight-colors||<fgcolor> <bgcolor> [<attribute> ...]||black yellow bold||This configuration command specifies the highlighting colors when searching for text from the article view.||search-highlight-colors white black bold
-searchresult-title-format||<format>||"%N %V - Search results (%u unread, %t total)%?F? matching filter '%F'&?" (localized)||Format of the title in search result. See "Format Strings" section of Newsboat manual for details on available formats.||searchresult-title-format "Search result"
+searchresult-title-format||<format>||"%N %V - Search results:\"%s\" (%u unread, %t total)%?F? matching filter '%F'&?" (localized)||Format of the title in search result. See "Format Strings" section of Newsboat manual for details on available formats.||searchresult-title-format "Search result"
```

Comment:
```suggestion
searchresult-title-format||<format>||"%N %V - Search results for '%s' (%u unread, %t total)%?F? matching filter '%F'&?" (localized)||Format of the title in search result. See "Format Strings" section of Newsboat manual for details on available formats.||searchresult-title-format "Search result"
```

I think this reads better, and the quotes are consistent with what we use for the filter.